db/migrate/20230611183949_create_api_keys.rb

```diff
@@ -0,0 +1,13 @@
+class CreateApiKeys < ActiveRecord::Migration[7.0]
+  def change
+    create_table :api_keys do |t|
+      t.integer :bearer_id, null: false
+      t.string :bearer_type, null: false
+      t.string :token, null: false
```

Comment:
Should we use store hashed tokens right from the beginning. Does not seem to be too much effort looking at https://keygen.sh/blog/how-to-implement-api-key-authentication-in-rails-without-devise/.